src/v2/Components/Artwork/Details.tsx

```diff
@@ -301,6 +310,7 @@ export const DetailsFragmentContainer = createFragmentContainer(Details, {
       href
       title
       date
+      is_saved: isSaved
```

Comment:
**suggestion**: These aliases were useful when we were migrating from Metaphysics's V1 to V2 schema but for new additions, we should stick with camelCased attribute names.